### ISSUE TITLE
SmartCity-Accessibility #158 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#113]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 134 lines...]
-- License info: Single-user license
-- Current date: Dec 29, 2016
JxMaps license valid.
Tests run: 2, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 2.518 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 5, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 9.24 sec - in smartcity.accessibility.navigation.NavigationTest
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f361638fbc8, pid=7379, tid=0x00007f35e9815700
#
# JRE version: OpenJDK Runtime Environment (8.0_111-b14) (build 1.8.0_111-8u111-b14-2ubuntu0.16.04.2-b14)
# Java VM: OpenJDK 64-Bit Server VM (25.111-b14 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libjxmaps-common-bd0a51c3-5ac0-4179-b40a-588c015f9c33.so+0xdbc8]  jxb::ipc::Event::reset()+0x28
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /var/lib/jenkins/workspace/SmartCity-Accessibility/hs_err_pid7379.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Aborted (core dumped)

Results :

Tests run: 39, Failures: 0, Errors: 0, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:04 min
[INFO] Finished at: 2016-12-29T11:55:38+00:00
[INFO] Final Memory: 28M/200M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: Error occurred in starting fork, check output in log -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Arthur Sapozhnikov] 77da4627791ee1752e442803822abd8fe4aaaa5b - Added a func for the enum

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/158/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
